stable/node-local-dns/README.md

```diff
@@ -12,44 +12,19 @@ Further documentation is [here](https://kubernetes.io/docs/tasks/administer-clus
 
 This helm chart works for both kube-proxy setups (iptables or ipvs).
 
-## How to install this chart
```

Comment:
This can't be removed 🙂

Reply:
Something must have gone wrong during the doc generation, it should look a bit better now